external/fypp/README.rst

```diff
@@ -199,7 +199,7 @@ repository::
 
   git clone https://github.com/aradi/fypp.git
 
-and check out the `master` branch.
+and check out the default main branch.
```

Comment:
As far as I know, there is only one default branch in `git`, but I might be mistaken. Therefore, default branch should be sufficient.